test/test_config/https/README.md

```diff
@@ -1,4 +1,4 @@
-To regenerate certificates, use [this](https://github.com/conjurdemos/dap-intro/tree/main/tools/simple-certificates)
+To regenerate certificates, use [this](https://github.com/conjurdemos/conjur-intro/tree/main/tools/simple-certificates)
```

Comment:
@royico I get 404 when i try to reach https://github.com/conjurdemos/conjur-intro/tree/main/tools/simple-certificate
does it work for you?

Reply:
@InbalZilberman , you have mistype there ( final s )
https://github.com/conjurdemos/conjur-intro/tree/main/tools/simple-certificates